test/Writer/FeedTest.php

```diff
@@ -654,12 +638,11 @@ public function testSetsGeneratorThrowsExceptionOnInvalidVersionDeprecated()
     public function testSetsGeneratorThrowsExceptionOnInvalidUriDeprecated()
     {
         $this->markTestIncomplete('Pending Laminas\URI fix for validation');
```

Comment:
out of scope,  but maybe this is already fixed, and we can remove above?

Reply:
No we can not, because there are no tests for `Laminas\Feed\Uri` and so we know nothing.

Example:

```php
var_dump(Laminas\Feed\Uri::factory('notauri')->isValid()); // true
```